client/blocks/product-purchase-features/product-purchase-features-list/business-onboarding.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import PurchaseDetail from 'components/purchase-detail';
+
+function trackOnboardingButtonClick() {
+	analytics.tracks.recordEvent( 'calypso_plan_features_onboarding_click' );
```

Comment:
Optional, but we can avoid a direct analytics lib call ( this makes testing components easier) by passing through a click event:

in business-onboarding.jsx
```javascript
import { noop } from 'lodash'; 
export default localize( ( { translate, onClick = noop } ) => {
    //...
     <PurchaseDetail onClick={ onClick }
   //... 
}
```

In the parent component:
```javascript
import { partial } from 'lodash';

import { recordTracksEvent } from 'state/analytics/actions';
//...

render() {
    //...
    			<BusinessOnboarding
				key="businessOnboarding"
                                onClick={ recordBusinessOnboardingClick }
			/>,
    //...
}

//... 
export default connect(
	( state ) => {
		//...
	},
        {
             recordBusinessOnboardingClick: partial( recordTracksEvent, 'calypso_plan_features_onboarding_click' )
        }
)( ProductPurchaseFeaturesList );
```